sky/serve/service_spec.py

```diff
@@ -139,6 +176,21 @@ def from_yaml_config(config: Dict[str, Any]) -> 'SkyServiceSpec':
             service_config['downscale_delay_seconds'] = policy_section.get(
                 'downscale_delay_seconds', None)
 
+            service_config['spot_placer'] = policy_section.get(
+                'spot_placer', None)
+            service_config['autoscaler'] = policy_section.get(
+                'autoscaler', None)
+            service_config['spot_zones'] = policy_section.get(
+                'spot_zones', None)
+            service_config['num_overprovision'] = policy_section.get(
+                'num_overprovision', None)
+            service_config['init_replicas'] = policy_section.get(
+                'init_replicas', None)
+            service_config['min_on_demand_replicas'] = policy_section.get(
+                'min_on_demand_replicas', 0)
```

Comment:
nit: Use None and set default value in ctor

Reply:
updated